terraform/environments/bootstrap/delegate-access/iam.tf

```diff
@@ -179,6 +179,166 @@ resource "aws_iam_policy" "member-access" {
   policy      = data.aws_iam_policy_document.member-access.json
 }
 
+module "instance-scheduler-access" {
+  count  = local.account_data.account-type == "member" && terraform.workspace != "testing-test" ? 1 : 0
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-cross-account-access?ref=v2.2.0"
+  providers = {
+    aws = aws.workspace
+  }
+  account_id             = local.modernisation_platform_account.id
+  additional_trust_roles = [format("arn:aws:iam::%s:role/github-actions", local.environment_management.account_ids[terraform.workspace])]
+  policy_arn             = aws_iam_policy.instance-scheduler-access[0].id
+  role_name              = "InstanceSchedulerAccess"
+}
+
+#tfsec:ignore:aws-iam-no-policy-wildcards
+data "aws_iam_policy_document" "instance-scheduler-access" {
```

Comment:
This seems like quite a lot of permissions, would it not only need describe and start stop EC2 permissions?

Reply:
It also needs permissions to read tags if we want to pick instances for starting / shutting down based on tags. I'd thought the following permissions would be appropriate:
```
ec2:DescribeInstances
ec2:DescribeInstanceStatus
ec2:DescribeTags
ec2:StartInstances
ec2:StopInstances
```

Reply:
Added "ec2:DescribeInstanceStatus",
            "ec2:DescribeTags"